### PR TITLE
feat: refactoring balances prices store

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by kelsos** [comment]
@lukicenturi do you think exchange rates would also make sense to be part of the prices store?

**Comment by lukicenturi** [comment]
@kelsos updated

File: frontend/app/src/store/balances/prices.ts [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+      const { taskId } = await api.getFiatExchangeRates(
+        currencies.map(value => value.tickerSymbol)
+      );
+
+      const meta: TaskMeta = {
+        title: i18n.t('actions.balances.exchange_rates.task.title').toString(),
+        numericKeys: []
+      };
+
+      const { awaitTask } = useTasks();
```
Comment (kelsos) [inline frontend/app/src/store/balances/prices.ts:113]: Mostly nitpick but can we pull useTasks/useNotifications on the store level?

**Review by kelsos — APPROVED** [review]
lgtm

File: frontend/app/src/premium/premium-apis.ts [hunk 2]
```diff
@@ -114,15 +116,16 @@ export const adexApi = (): AdexApi => {
...
     byLocation: computed<Record<string, BigNumber>>(() => {
       return store.getters['balances/byLocation'];
     }),
     aggregatedBalances: computed<AssetBalanceWithPrice[]>(() => {
       return store.getters['balances/aggregatedBalances'];
     }),
-    exchangeRate: currency =>
-      computed(() => store.getters['balances/exchangeRate'](currency))
+    exchangeRate: (currency: string) =>
+      computed(() => get(exchangeRate(currency)) ?? One)
```
[the remark this answers is not in the document]
Comment (lukicenturi) [reply frontend/app/src/premium/premium-apis.ts:128]: I think we almost always use ?? One. Also in premium components, it doesn't check for null/undefined, so I check it here.

**Pull request merged**